manifest_staging/charts/gatekeeper/templates/namespace-post-install.yaml

```diff
@@ -41,6 +41,50 @@ spec:
           securityContext:
             {{- toYaml .Values.postInstall.securityContext | nindent 12 }}
 ---
+{{- range $extraNamespace := .Values.postInstall.labelNamespace.extraNamespaces }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: gatekeeper-update-namespace-label-{{ $extraNamespace }}
+  labels:
+    app: '{{ template "gatekeeper.name" $ }}'
+    chart: '{{ template "gatekeeper.name" $ }}'
+    gatekeeper.sh/system: "yes"
+    heritage: '{{ $.Release.Service }}'
+    release: '{{ $.Release.Name }}'
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
+spec:
+  template:
+    metadata:
+      labels:
+        app: '{{ template "gatekeeper.name" $ }}'
+        release: '{{ $.Release.Name }}'
+    spec:
+      restartPolicy: OnFailure
+      {{- if $.Values.postInstall.labelNamespace.image.pullSecrets }}
+      imagePullSecrets:
+      {{- $.Values.postInstall.labelNamespace.image.pullSecrets | toYaml | nindent 12 }}
+      {{- end }}
+      serviceAccount: gatekeeper-update-namespace-label
+      nodeSelector:
+        kubernetes.io/os: linux
+      containers:
+        - name: kubectl-label
+          image: "{{ $.Values.postInstall.labelNamespace.image.repository }}:{{ $.Values.postInstall.labelNamespace.image.tag }}"
+          imagePullPolicy: {{ $.Values.postInstall.labelNamespace.image.pullPolicy }}
+          args:
+            - label
+            - ns
+            - {{ $extraNamespace }}
```

Comment:
Can we add these namespaces as part of the `gatekeeper-update-namespace-label` job instead of creating a new job for each extra namespace?

Reply:
@chewong Done.